packages/@lwc/engine-core/src/framework/renderer.ts

```diff
@@ -43,3 +43,10 @@ export interface Renderer<N = HostNode, E = HostElement> {
     isConnected(node: N): boolean;
     tagName(element: E): string;
 }
+
+// This is a temporary workaround to get the @lwc/engine-server to evaluate in node without having
+// to inject at runtime.
+export const documentObject: Document = typeof document !== 'undefined' ? document : ({} as any);
```

Comment:
This needs to be fixed at some point for the `@lwc/engine-server` to handle AOM and global HTML attributes.